app/src/main/java/org/oppia/android/app/onboarding/IntroActivityPresenter.kt

```diff
@@ -15,7 +16,7 @@ import javax.inject.Inject
 private const val TAG_LEARNER_INTRO_FRAGMENT = "TAG_INTRO_FRAGMENT"
 
 /** Argument key for bundling the profile nickname. */
-const val PROFILE_NICKNAME_ARGUMENT_KEY = "IntroFragment.Arguments"
+const val INTRO_FRAGMENT_ARGUMENT_KEY = "IntroFragment.Arguments"
```

Comment:
Since you’ve touched this code, would it be possible to consider moving the IntroFragment creation and key into the IntroFragment itself?

Reply:
Done